### PR TITLE
Dockerfile: rebase on ubi8-minimal

### PR DESCRIPTION
**Pull request by fromanirh**
See: https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,8 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 MAINTAINER "Francesco Romani" <fromani@redhat.com>
 ENV container docker
 
-RUN yum -y update
```
Comment (ksimon1) [inline Dockerfile]: these images doesn't need update?

Comment (fromanirh) [reply Dockerfile]: Not sure. AFAIU they are supposed to always be up to date. Anyway, the update mechanism on -minimal is going to be different, so this should be changed anyway.
I plan to drop the update and to reintroduce it later if needed.

Comment (ksimon1) [reply Dockerfile]: According to that article, it should be ok to drop update command.

Comment (fromanirh) [reply Dockerfile]: @ksimon1 worst case I'll reintroduce it again later :)

**Pull request merged**